src/flypad-backend/src/Aircraft/AircraftProcedures.h

```diff
@@ -3,7 +3,10 @@
 #include <map>
 #include <string>
 #include <vector>
+#include <algorithm>
```

Comment:
Can you cleanup the includes here? For example map is not used, etc.

Reply:
AFAIK map was already included, but I'll take a look

Reply:
Yes. But if you already are changing the codestyle I think a little cleanup doesn't hurt